packages/pigeon/platform_tests/ios_unit_tests/ios/RunnerTests/MultipleArityTest.m

```diff
@@ -16,7 +16,7 @@ @implementation MultipleAritytest
 
 - (void)testSimple {
   HandlerBinaryMessenger *binaryMessenger =
-      [[HandlerBinaryMessenger alloc] initWithCodec:MultipleArityHostApiGetCodec()
+      [[HandlerBinaryMessenger alloc] initWithCodec:[FlutterStandardMessageCodec sharedInstance]
```

Comment:
These should be a call to the GetCodec function no?

Reply:
If you think so, I was just continuing the current design language.

Reply:
Maybe I'm missing something here, but where is this 'GetCodec' function?

Reply:
It should be generated.  Like if I generate `//pigeons/messages.dart` for objc, the following functions is generated:

```objc
NSObject<FlutterMessageCodec> *ACMessageFlutterSearchApiGetCodec(void);
```

for:
```dart
@FlutterApi()
abstract class MessageFlutterSearchApi {
  MessageSearchReply search(MessageSearchRequest request);
}
```

Reply:
oh, you mean, they should stay the way they were before?

Reply:
The getCodec functions were removed is the issue. I'll look into reverting that